src/main/java/hq/CaseAPIs.java

```diff
@@ -30,7 +32,10 @@
  */
 public class CaseAPIs {
 
-    public static UserSqlSandbox forceRestore(String username, String domain, String xml) throws Exception {
+    public static UserSqlSandbox forceRestore(RestoreFactory restoreFactory) throws Exception {
+        String domain = restoreFactory.getDomain();
+        String username = TableBuilder.scrubName(restoreFactory.getUsername());
```

Comment:
I think for sanity/debugging it'll be good to have this table has the asUsername as well so we can tell if the db is a restore as db